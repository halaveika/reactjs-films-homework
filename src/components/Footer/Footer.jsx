import React from 'react';
import { Layout } from 'antd';
import './Footer.scss';

export default function Footer() {
  return (
    <Layout.Footer className="footer">
      <span className="title">films</span>
      <span className="copyright">COPYRIGHT Â© 2021 Films  HALAVEIKA ALIAKSANDR</span>
    </Layout.Footer>
  );
}
