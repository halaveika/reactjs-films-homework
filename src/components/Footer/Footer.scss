.footer {
  display: flex;
  flex-direction: row;
  flex-wrap: wrap;
  justify-content: center;
  align-items: center;
  height: 120px;
  width: 100vw;
  position: relative;
  padding: 20px;

  & span {
    display: block;
    color: #8a8c8d;
    margin-right: 5px;

    & a {
      color: #8a8c8d;
      text-decoration: none;
    }
  }

  .title {
    width: 100%;
    display: block;
    text-transform: uppercase;
    color: #1890ff;
    text-align: center;
    cursor: pointer;
  }
}
