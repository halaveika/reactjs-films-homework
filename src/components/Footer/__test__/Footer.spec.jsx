import React from 'react';
import { create } from 'react-test-renderer';
import Footer from '..';

describe('test Footer component', () => {

  it('should render Footer component', () => {
    const component = create(
          <Footer/>,
      );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

});
