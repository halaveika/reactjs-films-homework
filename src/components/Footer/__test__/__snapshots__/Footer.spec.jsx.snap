// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Footer component should render Footer component 1`] = `
<footer
  className="ant-layout-footer footer"
>
  <span
    className="title"
  >
    films
  </span>
  <span
    className="copyright"
  >
    COPYRIGHT Â© 2021 Films  HALAVEIKA ALIAKSANDR
  </span>
</footer>
`;
